### DISCUSSION TITLE
HTTP 401 Unauthorized

### DISCUSSION THREAD

**Discussion by SilvandroOliveira** [keycloak/keycloak#21213] [Q&A]
I'm trying to make a request to list all users from the REST API, but it's returning the HTTP 401 Unauthorized error, I would like to know if I need to do any extra steps. Initially I generated the access_token with the client_id "admin-cli" and grant_type "password", and I sent this token in the "Authorization" header of the request with the value "Bearer token". I use Keycloak version 21.1.1.

**Comment by rranjangupta** [comment]
how did you solve this issue ?
